cmd/poseidon/poseidon.go

```diff
@@ -31,6 +31,11 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	Interval = 2 * time.Second
+	TimeOut  = 10 * time.Minute
```

Comment:
nit: Timeout

And what interval? what timeout?

Reply:
OK, I have updated.